assets/js/main.js

```diff
@@ -1,55 +1,100 @@
+  );
+  const selectedCityLocation = await fetchedData.json();
+
   
-    const time = document.getElementById("time");
-    const now = new Date();
-    const tomorrowDate = new Date(list[5].dt *1000);
-    const tomorrowDayName = document.getElementById("tomorrow-day-name");
-    const dayNameTomorrow = tomorrowDate.toLocaleString("en-us", { weekday: "long" });
-    tomorrowDayName.innerHTML = dayNameTomorrow;
-    const dayAfterTomorrowDate = new Date(list[11].dt *1000);
-    
-    const dayAfterTomorrowDayName = document.getElementById("day-after-tomorrow-day-name");
-    const DayNameDayAfterTomorrow = dayAfterTomorrowDate.toLocaleString("en-us", { weekday: "long" });
-    dayAfterTomorrowDayName.innerHTML = DayNameDayAfterTomorrow;
-    
-    const year = now.getFullYear();
-    const month = now.toLocaleString("default", { month: "short" });
-    const dayName = now.toLocaleString("en-us", { weekday: "long" });
-    const dayNumber = now.getUTCDate();
-    const date = `${dayName}, ${dayNumber} ${month} ${year}`;
-    time.innerHTML = date;
+  const [{lat,lon}] = selectedCityLocation;
+
+  getWeather(lat, lon);
+}
+
+async function getWeather(lat, lon) {
+  const result = await fetch(
+    `https://api.openweathermap.org/data/2.5/forecast?lat=${lat}&lon=${lon}&units=metric&appid=${appId}`
+  );
+  const { city, list } = await result.json();
+  const cityName = document.getElementById("city-name");
+  cityName.innerHTML = city.name;
+
+  const wind = document.getElementById("wind");
+  const windSpeed = list[0].wind.speed;
+  wind.innerHTML = windSpeed;
+
+  const humidity = document.getElementById("humidity");
+  const humidityPercent = list[0].main.humidity;
+  humidity.innerHTML = `${humidityPercent}%`;
+
+  const temp = document.getElementById("temp");
+  const todayTemp = document.getElementById("today-temp");
+  const tomorrow = document.getElementById("tomorrow");
+  const dayAfterTomorrow = document.getElementById("day-after-tomorrow");
+  const temperature = list[0].main.temp;
+  temp.innerHTML = `${Math.round(temperature)}&#176;C`;
+  todayTemp.innerHTML = `${Math.round(temperature)}&#176;C`;
+  const tomorrowTemperature = list[5].main.temp;
+  tomorrow.innerHTML = `${Math.round(tomorrowTemperature)}&#176;C`;
+  const dayAfterTomorrowTemp = list[10].main.temp;
+  dayAfterTomorrow.innerHTML = `${Math.round(dayAfterTomorrowTemp)}&#176;C`;
+
+  const description = document.getElementById("description");
+  const descrip = list[0].weather[0].description;
+  description.innerHTML = descrip;
+
+  const iconId = document.getElementById("current-icon");
+  const icon = list[0].weather[0].icon;
+  const iconImage = `http://openweathermap.org/img/wn/${icon}@2x.png`;
+  iconId.innerHTML = `<img src=${iconImage} />`
+
+  const todayIcon = document.getElementById("today-icon");
+  todayIcon.innerHTML = `<img src=${iconImage} />`
+
+  const tomorrowIcon = document.getElementById("tommorow-icon");
```

Comment:
src is the same as today's icon. so you are getting the same icons for all days. please fix this.

Reply:
Fixed.